puppeteer/setup-drupal.ts

```diff
@@ -38,7 +38,7 @@ import puppeteer from "puppeteer";
 
   await h5pBildetemaEditorIframe.type("#field-extratitle--1", title);
   await editorIframe.click("#edit-submit");
-  await page.waitForNavigation({ waitUntil: "networkidle2" });
+  await page.waitForTimeout(5000);
```

Comment:
Can we make use of [`page.waitForSelector`](https://puppeteer.github.io/puppeteer/docs/puppeteer.page.waitforselector/)?

Reply:
Ive tried to use waitForSelector but i got some weird errors...
`[30368:259:0616/101156.127187:ERROR:skia_output_device_buffer_queue.cc(354)] Invalid mailbox.
[30368:259:0616/101156.127193:ERROR:shared_image_manager.cc(277)] SharedImageManager::ProduceOverlay: Trying to Produce a Overlay representation from a non-existent mailbo`